tests/Functional/CorporateFooterRenderTest.php

```diff
@@ -0,0 +1,435 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_theme\Functional;
+
+use Drupal\Tests\BrowserTestBase;
+use Symfony\Component\Yaml\Yaml;
+use Drupal\Component\Render\FormattableMarkup;
+use Behat\Mink\Element\NodeElement;
+
+/**
+ * Test footer block rendering.
+ */
+class CorporateFooterRenderTest extends BrowserTestBase {
+
+  /**
+   * The config factory.
+   *
+   * @var \Drupal\Core\Config\ConfigFactoryInterface
+   */
+  protected $configFactory;
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
+    'block',
+    'page_header_metadata_test',
```

Comment:
Do we need this one here?

Reply:
did before, dono why, but indeed works well without.